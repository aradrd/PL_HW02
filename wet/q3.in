val test_1 = if alive = is_alive (alive, empty, empty) (empty, alive, alive) (empty, empty, empty) then "Passed" else "Failed";
val test_2 = if empty = is_alive (alive, alive, alive) (empty, alive, empty) (alive, alive, alive) then "Passed" else "Failed";
