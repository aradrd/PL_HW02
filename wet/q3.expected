Standard ML of New Jersey (64-bit) v110.99.2 [built: Sun Mar 20 20:10:23 2022]
[opening hw2_q3.sml]
datatype cell = alive | empty
val is_alive = fn :
  cell * cell * cell -> cell * cell * cell -> cell * cell * cell -> cell
- val test_1 = "Passed" : string
val test_2 = "Passed" : string
- 
